README.md

```diff
@@ -6,15 +6,17 @@
 </p>
 <br>
 
-A white paper specifying an über-shader that aims to provide a material representation capable of accurately modeling the vast majority of materials used in practical visual effects and feature animation productions.
 
-OpenPBR is an open standard hosted by the [Academy Software Foundation](https://www.aswf.io/) (ASWF), and is organized as a subproject of [MaterialX](https://materialx.org/).
+OpenPBR Surface is a specification of a surface shading model intended as a standard for computer graphics. It aims to provide a material representation capable of accurately modeling the vast majority of CG materials used in practical visual effects and feature animation productions.
 
+OpenPBR Surface is an open standard hosted by the [Academy Software Foundation](https://www.aswf.io/) (ASWF), and is organized as a subproject of [MaterialX](https://materialx.org/).
 
-### Resources
+### Specification
 
-* **[Specification](https://academysoftwarefoundation.github.io/OpenPBR/)**
+* **[White paper](https://academysoftwarefoundation.github.io/OpenPBR/)**
```

Comment:
This line reverts the changes from #182. Is this intentional or an oversight?

Reply:
I think it's actually better like this, as under the "Specification" heading we have a number of components, i.e. the white paper, the parameter reference, the MaterialX graph, and the citation.

We say now "OpenPBR surface is a specification" (rather than e.g. "A white paper specifying" as for standard surface). But there is a white paper as part of that specification. Hopefully it makes sense..

Reply:
Oh ok, I see the logic.